### PR TITLE
chore: use crawler

### PR DESCRIPTION
**Pull request by stefano-tech-ai**

### CODE REVIEW

File: index-v1.yaml
Comment (kesara) [inline index-v1.yaml:1]: @stefanomunarini What is the use of this `index-v1.yaml` and `index-v1.zip` files?

Comment (stefano-tech-ai) [reply index-v1.yaml:1]: Hi @kesara , these are indexes auto-generated by the new crawler. @andrew2net maybe can provide further explanation on their usage?

Comment (andrew2net) [reply index-v1.yaml:1]: We use index files to search documents. Relaton gems download `index-v1.zip`, unzip, and use it to resolve documents' filenames.
`index-v1.yaml` is unzipped version of `index-v1.zip`. It's useful as list of references for users.

Comment (kesara) [reply index-v1.yaml:1]: Then we don't need to keep the zip file checked in the repository correct?

Comment (stefano-tech-ai) [reply index-v1.yaml:1]: @kesara that's correct. I've submitted a [PR](https://github.com/relaton/support/pull/16) to the relaton/support crawler workflow in which I made it possible to specify whether index files should be committed or not. 

I'll update this PR as soon as the other PR is approved and merged

Comment (stefano-tech-ai) [reply index-v1.yaml:1]: PR is now ready for review @kesara , thanks

**Review by kesara — APPROVED** [review]

**Comment by kesara** [comment]
@stefanomunarini I can not push to your branch to resolve the conflict. Can you resolve it? ta.

**Comment by stefanomunarini** [comment]
> @stefanomunarini I can not push to your branch to resolve the conflict. Can you resolve it? ta.

Conflict in flag.txt is fixed. However, if the workflow runs again and updates this file, a new conflict will be generated. PR should be merged before the next workflow runs @kesara

**Comment by stefano-tech-ai** [comment]
@kesara conflict resolved

**Comment by kesara** [comment]
> @kesara conflict resolved

Thank you!

**Pull request merged**